cmd/tools/rest/href_builder_test.go

```diff
@@ -0,0 +1,101 @@
+package rest
+
+import (
+	"github.com/google/go-cmp/cmp"
+	"strconv"
+	"testing"
+)
+
+func TestBuild(t *testing.T) {
+	testQuery := "storage/volumes"
+	testFields := []string{"name", "svm"}
+	testHiddenFields := []string{"statistics"}
+	testFilter := []string{""}
+	testReturnTimeout := 10
+	isIgnoreUnknownFieldsEnabled := true
+	expectedHrefTest1 := "api/storage/volumes?return_records=true&fields=name,statistics,svm&return_timeout=10&ignore_unknown_fields=true"
+	hrefTest1 := NewHrefBuilder().
+		APIPath(testQuery).
+		Fields(testFields).
+		HiddenFields(testHiddenFields).
+		Filter(testFilter).
+		ReturnTimeout(&testReturnTimeout).
+		IsIgnoreUnknownFieldsEnabled(isIgnoreUnknownFieldsEnabled).
```

Comment:
isIgnoreUnknownFieldsEnabled is always true so this can be changed to
`IsIgnoreUnknownFieldsEnabled(true).`